packages/pymedphys_dicom/src/pymedphys_dicom/dicom/dose.py

```diff
@@ -42,6 +42,15 @@
 # pylint: disable=C0103
 
 
+def zyx_and_dose_from_dicom(dicom_filepath):
+    ds = pydicom.dcmread(dicom_filepath, force=True)
```

Comment:
@Centrus007 the new coordinate functions happened to break `gamma_dicom`. It was unfortunate the the testing file had x and y coords of the same length